source/includes/_errors.md

```diff
@@ -14,6 +24,7 @@ Error Code | Meaning
 410 | Gone -- The kitten requested has been removed from our servers
 413 | Request Entity Too Large -- You sent too much data to us
 418 | I'm a teapot
+422 | Unprocessable Entity
```

Comment:
422 is WebDav, which is around from some time now, but it's still an extension to HTTP and we're not using WebDav in this context. Even more they have been rephrasing the 400 status description in the [new RFC](https://tools.ietf.org/html/rfc7231#page-58), making it less related to syntax errors.
This said, I think we can discuss for hours around this topic in front of some :beers:, but I'll make the call for `400` because of consistency with the other responses.